### DISCUSSION TITLE
Unable to run NanoMQ on Windows Server 2022 OS

### DISCUSSION THREAD

**Discussion by anandbwfrd** [nanomq/nanomq#1172] [Q&A] [answered]
I am getting 2 errors when trying to run NANOMQ using default configuration on Server 2022 OS.

Can someone help me with this issue ASAP please?

I have attached the screenshot of the errors that I am getting.

thanks,
Anand.
![Screenshot 2023-04-17 121755](https://user-images.githubusercontent.com/112025627/232565537-4d0239d6-b745-4278-8818-9250ac38ffe9.png)
![Screenshot 2023-04-17 121741](https://user-images.githubusercontent.com/112025627/232565538-cbbed0a3-ff77-4471-9155-9bf37d34d477.png)

**Comment by JaylinYu** [comment, marked as answer]
[dll.zip](https://github.com/emqx/nanomq/files/11256901/dll.zip)
While we replace the version, you can try to put those dlls file in your C:/Windows/system32/ folder

**Comment by anandbwfrd** [comment]
Okay will add those to the folder that you have given and try it out.

Thanks Jaylin!

From: Jaylin ***@***.***>
Sent: Monday, April 17, 2023 7:56 PM
To: emqx/nanomq ***@***.***>
Cc: Balasubramanian, Anand ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [emqx/nanomq] Unable to run NanoMQ on Windows Server 2022 OS (Discussion #1172)


CAUTION: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.

________________________________

dll.zip<https://github.com/emqx/nanomq/files/11256901/dll.zip>
While we replacing the version, you can try to put those dlls file in your C:/Windows/system32/ folder

-
Reply to this email directly, view it on GitHub<https://github.com/emqx/nanomq/discussions/1172#discussioncomment-5643051>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2WWAG7OHKYRIME2DPWUUQ3XBXRARANCNFSM6AAAAAAXBP5XVA>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

This electronic communication is for the use of the intended recipient(s) only, and may contain confidential, privileged or proprietary information. If you are not an intended recipient, please reply to the sender and then immediately delete and destroy all copies of the communication. See our Electronic Communications Terms and Conditions<https://www.weatherford.com/en/about-us/policies/terms-and-conditions-for-electronic-communications/> for further information
